examples/calendar/frontend/src/types.ts

```diff
@@ -0,0 +1,12 @@
+export interface Auth {
+	accessToken: string;
+}
+
+export interface User {
+	id: number,
+	username: string, 
+	publicAddress: string, 
+	nonce: string, 
+	createdAt: string, 
```

Comment:
I think we don't need createdAt and updatedAt in front-end

Reply:
Updated the code.